From e9eb525a6a1314d890fd0afe2e08aa81ce710605 Mon Sep 17 00:00:00 2001
From: maxwen <max.weninger@gmail.com>
Date: Tue, 2 Oct 2018 15:18:41 +0200
Subject: [PATCH] omni: add hours and minutes to ro.omni.version

so we no longer need to wait for next day on official rebuilds

Change-Id: Iff88cd729eb52d4891fd59bf9fb98a5da8693348
---
 config/version.mk | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/config/version.mk b/config/version.mk
index 97e5de9..aafdaa6 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -8,14 +8,10 @@ TARGET_PRODUCT_SHORT := $(TARGET_PRODUCT)
 TARGET_PRODUCT_SHORT := $(subst omni_,,$(TARGET_PRODUCT_SHORT))
 
 # Build the final version string
-ifdef BUILDTYPE_RELEASE
-    ROM_VERSION := $(PLATFORM_VERSION)-$(TARGET_PRODUCT_SHORT)
-else
 ifeq ($(ROM_BUILDTIME_UTC),y)
-    ROM_VERSION := $(PLATFORM_VERSION)-$(shell date -u +%Y%m%d)-$(TARGET_PRODUCT_SHORT)-$(ROM_BUILDTYPE)
+    ROM_VERSION := $(PLATFORM_VERSION)-$(shell date -u +%Y%m%d%H%M)-$(TARGET_PRODUCT_SHORT)-$(ROM_BUILDTYPE)
 else
-    ROM_VERSION := $(PLATFORM_VERSION)-$(shell date +%Y%m%d)-$(TARGET_PRODUCT_SHORT)-$(ROM_BUILDTYPE)
-endif
+    ROM_VERSION := $(PLATFORM_VERSION)-$(shell date +%Y%m%d%H%M)-$(TARGET_PRODUCT_SHORT)-$(ROM_BUILDTYPE)
 endif
 
 # Apply it to build.prop
-- 
2.20.1

